dist/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  <script defer src="main.js"></script></head>
+  <body>
+    <header class="header">
+      <h1>Leaderboard <i class="fa-solid fa-trophy"></i></h1>
+    </header>
+    <section class="container-inner">
+      <div class="main">
+        <div class="leaderboard">
+          <h2>Recent scores</h2>
+          <button id="refresh-btn">Refresh</button>
+        </div>
+        <div class="add">
+          <h2>add your score</h2>
+        </div>
+      </div>
+      <div class="score">
+        <ul class="scorelist"></ul>
+        <form action="" id="form-add">
+          <div>
+            <input type="text" name="name" placeholder="Your Name" />
```

Comment:
- [ ] Although styling is not required at this point, your wireframe needs to match the given wireframe.

Your design
![image](https://user-images.githubusercontent.com/81932478/208930171-7db780ea-6d2e-4d49-8151-9f4471060aa2.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/208930349-ca11f365-c0db-42fb-a1cb-24e9ad76fc45.png)